app/controllers/pages_controller.rb

```diff
@@ -37,6 +37,15 @@ def show
       breadcrumb page.title, page.path if @page.title.present?
     end
 
+    if @page.path.match?(/funding-your-training/)
```

Comment:
I would maybe call this out into its own route, feels wrong having it in the generic `pages#show` action